### PR TITLE
CVE Exclusions for false positives

### PR DESCRIPTION
**Pull request by Hildebrand-Ritense**
Added cve exlusions file that ignores a number of false positives until december '24.

Quoting @anna-ritense 
- log4j coming from dokka - we are already on the latest version so there’s nothing that can be done there without a new release of dokka (Theo [reported](https://github.com/Kotlin/dokka/issues/3508) this and it was adressed already in May) or removal of dokka altogether. tbd.
- guava coming from dokka - same as above, but remedied locally by overriding guava version to non-breaking version: 32.0.1
- apache commons compress coming from also dokka - same as above, but remedied by overriding version to 1.26.0 or higher

### CODE REVIEW

File: .java-version [hunk 1]
```diff
@@ -1 +1 @@
-system
+21
```
Comment (anna-ritense) [inline .java-version:1]: Can this be kept system?
jvm fails if I only have 21.0.1 installed on my system.

Comment (Hildebrand-Ritense) [reply .java-version:1]: No sorry, that defeats the whole purpose of using a .java-version file. Don't you use sdkman or jenv? I can have a look with you.

Comment (Hildebrand-Ritense) [reply .java-version:1]: (If you mean jenv can't find a matching version for 21, use this to add a symlink for global 21: `ln -s /Users/<username>/Library/Java/JavaVirtualMachines/temurin-21.0.1/Contents/Home ~/.jenv/versions/21`)

Comment (anna-ritense) [reply .java-version:1]: Adding a symlink is a solution for me, but not for any random person checking out the repo. I guess that's something to figure out as we go.

**Review by anna-ritense — APPROVED** [review]

**Pull request merged**